graph/db/graph.go

```diff
@@ -119,6 +88,13 @@ func (c *ChannelGraph) Start() error {
 	log.Debugf("ChannelGraph starting")
 	defer log.Debug("ChannelGraph started")
 
+	if c.graphCache != nil {
+		if err := c.populateCache(); err != nil {
+			return fmt.Errorf("could not populate the graph "+
```

Comment:
nit: the started flag remains dirty, but i think that's okay since we likely won't try to start the same graph instance again.